src/app/help/en/faq.md

```diff
@@ -817,6 +818,30 @@ No. None of these actions incur any fees.
 
 A transfer of tokens between accounts typically takes 3 seconds. This is far faster than most blockchain tokens.
 
+<a href="#Table_of_Contents_Economics">^</a>
+## <span id="How_does_bandwidth_work">How does bandwidth work?</span>
+
+Bandwidth represents a share in the resources of the Steem blockchain's network and it is what enables users to perform actions like posting, voting, commenting and making token transfers. Every action a user takes on Steemit, with the exception of viewing content, consumes part of his/her account's bandwidth allocation. All accounts are allocated bandwidth according to how much Steem Power the account holds. The more the Steem Power, the more bandwidth is allocated.
+
+This bandwidth rate-limiting is employed to safeguard the blockchain from spam attacks.
+
+To see how much bandwidth your account has at any time, visit [https://steemd.com/](https://steemd.com)@your_username and look for the section shown in the screenshot below.
+
+![Screenshot (3).png](https://steemitimages.com/DQmYwLpTW1RVY1jUKwLwEW9QSbEnP9DRW9h2BCerk3PfEDj/Screenshot%20(3).png)
+
+Your account's bandwidth is measured in bytes, kilobytes, megabytes or gigabytes, and it recharges gradually over time. The progress bar labelled "bandwidth remaining" shows how much bandwidth you have left as a percentage.
+
+### What happens when I run out of bandwidth?
+
+Normally, most active accounts will have enough bandwidth to perform the usual actions on Steemit, but when there are a lot of people using the platform at a certain time, bandwidth allocations decrease as the bandwidth is spread amongst all active accounts. Most accounts will still have more than enough bandwidth to use Steemit, but exceptions exist. Accounts with very low amounts of Steem Power are more susceptible to bandwidth problems.
+
+If your account runs out of bandwidth, you will not be able to perform actions on Steemit. To increase your bandwidth allocation, you can:
+
+- Increase your Steem Power **(best option)**
+- Wait until your bandwidth recharges
+
+*To increase your Steem Power, you can buy more STEEM and power up, receive STEEM or Steem Power from someone else or have someone delegate some Steem Power to your account.*
```

Comment:
Technically if a user is out of bandwidth and they receive STEEM, they will be unable to power it up.

Recommend:
To increase your Steem Power, you can buy more Steem Power through a third-party exchange [BlockTrades](https://blocktrades.us/?input_coin_type=eth&output_coin_type=steem_power&receive_address=), receive Steem Power from someone else, or have someone delegate Steem Power to your account.

Reply:
Another good point. I'll make the changes.